nuxeo-coldstorage/src/main/java/org/nuxeo/coldstorage/operations/MoveToColdStorage.java

```diff
@@ -76,12 +76,13 @@ public DocumentModel run(DocumentModel document) {
 
     @OperationMethod
     public DocumentModelList run(DocumentModelList documents) {
-        DocumentModelList result = new DocumentModelListImpl();
+        DocumentModelList result = null;
```

Comment:
The changes in this file are unrelated to the ticket being fixed, please do a separate ticket/commit.